packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -696,7 +732,8 @@ void main() {
           home: SelectableRegion(
             focusNode: focusNode,
             selectionControls: materialTextSelectionControls,
-            child: const Column(
+            child: const Flex(
```

Comment:
Instead of changing this, I will probably update the expected copied text

Reply:
I think the issue here is the new Row/Column are interfering with the selection detection in these tests

Reply:
No entirely sure what do you mean? The finder below doesn't seem to care about column/row

Reply:
Not at my computer right now to give actual logs, but if you run the tests without these changes you should see the failure.

Reply:
Right, it throws range errors because the selections aren't found

```console
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following RangeError was thrown running a test:
RangeError (index): Invalid value: Valid value range is empty: 0

When the exception was thrown, this was the stack:
#0      List.[] (dart:core-patch/growable_array.dart:264:36)
#1      main.<anonymous closure>.<anonymous closure> (file:///Users/rexios/repos/flutter/packages/flutter/test/widgets/selectable_region_test.dart:1187:35)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)

The test description was:
  can copy from toolbar
```

Reply:
It might just be that the detection logic needs fixed, but I don't know enough about it to do it easily

Reply:
Thanks! All the tests should pass now.